plat/qemu/qemu_image_load.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017, Linaro Limited. All rights reserved.
```

Comment:
Correct. The "ARM Limited and Contributors" boilerplate code should be used for all contributions except imported 3rd party code (which this isn't). That teesmc_opteed.h header is a bit special as it contains code from a Linaro maintained project (OPTEE), which was modified and contributed here.